### DISCUSSION TITLE
AWS yardımı nasıl alınıyor?

### DISCUSSION THREAD

**Discussion by LatifY** [acikkaynak/deprem-yardim-projesi#47] [Q&A]
Uygulamam için AWS yardımı almak istiyorum. Duyduğuma göre depremle ilgili uygulama geliştirenlere kredi desteği sağlanıyormuş

Projem: www.zelzele.live

**Comment by yamac-kurtulus** [comment]
> Uygulamam için AWS yardımı almak istiyorum. Duyduğuma göre depremle ilgili uygulama geliştirenlere kredi desteği sağlanıyormuş
> 
> Projem: [www.zelzele.live](http://www.zelzele.live)

Discord'a girdiniz mi? Oradan yönlendirmeleri okuyup moderatör arkadaşlarla iletişime geçip gerekli desteği alabilirsiniz.

**Comment by ErtanTaner** [comment]
Bana yazarsan discord üzerinden sana bilgilendirme yapabilirim. Cloud ekibinden değilim ama cloud ekibinden arkadaşları da yönlendirebilirim. İsmim Ertan Taner discord sunucusunda.

**Comment by ramazansancar** [comment]
Merhaba arkadaşlar,
Öncelikle paylaşımınız için teşekkürler. Latif ile iletişime geçilerek süreç başlatılmış.

**Reply by ErtanTaner** [reply]
Ben Ramazan ile şahsi olarak iletişime geçip projesine dahil olmak istiyorum. O yüzden de yazmıştım.
